recipe/meta.yaml

```diff
@@ -1,21 +1,21 @@
 {% set name = "mnelab" %}
-{% set version = "0.8.6" %}
+{% set version = "0.9.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 1cd94b74e9f41a69f08ee34bbe15aefe543c780ac7ac8571eab599529c0ac970
+  sha256: 3924be68d097712bb5297c23e2a0bd859a67628a3e7d37417f8fd1e085a7cd75
 
 build:
   number: 0
   script: {{ PYTHON }} -m pip install . -vv
   entry_points:
     - mnelab = mnelab.__main__:main
   osx_is_app: true
-  skip: true  # [py<38]
+  skip: true  # [py<39]
```

Comment:
For some versions of MNE yes. But I think without this it would still build the recipe, just build it against and be installable with mne 1.6 for example.

I think it's better for each recipe to reflect its own dependencies / pins directly anyway rather than implicitly rely on those of its dependencies